pkg/services/provisioning/datasources/config_reader_test.go

```diff
@@ -326,9 +320,20 @@ func validateDatasource(t *testing.T, dsCfg *configs) {
 }
 
 func validateDatasourceV1(t *testing.T, dsCfg *configs) {
-	validateDatasource(t, dsCfg)
+	validateDatasourceV0(t, dsCfg)
```

Comment:
I just wanted to clean up the names. `validateDatasource` validates version=0 of the provisioning file structure. V1 calls V0 because it has to be backwards compatible with V0.